gravity-forms/gw-update-posts.php

```diff
@@ -97,9 +97,13 @@ public function update_post_by_entry( $entry, $form ) {
 		}
 
 		if ( $this->_args['post_date'] ) {
-			$new_date_time       = $this->get_post_date( $entry, $form );
-			$post->post_date     = $new_date_time;
-			$post->post_date_gmt = get_gmt_from_date( $new_date_time );
+			$new_date_time = $this->get_post_date( $entry, $form );
```

Comment:
Would it make more sense to return `null` in `get_post_date` if these aren't set?